src/jaxoplanet/experimental/starry/rotation.py

```diff
@@ -201,12 +201,10 @@ def dotR(l_max: int, u: Array) -> Callable[[Array], Array]:
         - theta is the rotation angle in radians
     """
     Rls = [Rl(l) for l in range(l_max + 1)]
-    n_max = l_max**2 + 2 * l_max + 1
 
-    @partial(jnp.vectorize, signature=f"({n_max},{n_max}),()->({n_max},{n_max})")
     def R(M: Array, theta: Array) -> Array:
         alpha, beta, gamma = axis_to_euler(u[0], u[1], u[2], theta)
-        return np.hstack(
+        return jnp.hstack(
```

Comment:
There was a regular numpy call which meant that this function couldn't be wrapped in a `jit`. @lgrcia